tasks/15_confirm-ending/confirm-ending.js

```diff
@@ -0,0 +1,4 @@
+export function confirmEnding(str, target) {
+  if (str.substring(str.length - target.length) === target) return true;
+  return false;
```

Comment:
Since `str.substring(str.length - target.length) === target` will return `true/false`, maybe we can simplify this to be 
```
return str.substring(str.length - target.length) === target;
```

Reply:
You are absolutely right, i adopted your suggestion.